2016/playbook/session_3/experiments/improve_social_behavior_of_hackers_with_card_games.md

```diff
@@ -0,0 +1,19 @@
+# Running card games during lunch
+
+## Implementation
+
+-Run a card game during lunch which initiates conversation and increases hacker productivity after lunch
```

Comment:
This needs a space after the dash to be a proper list in markdown. Looking at your other experiment file though, it seems like this shouldn't be a list.